packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -531,26 +566,142 @@ class CupertinoPopupSurface extends StatelessWidget {
   /// Some popups, like iOS's volume control popup, choose to render a blurred
   /// area without any white paint covering it. To achieve this effect,
   /// [isSurfacePainted] should be set to false.
+  ///
+  /// Defaults to true.
   final bool isSurfacePainted;
 
   /// The widget below this widget in the tree.
-  final Widget? child;
+  ///
+  /// Because [CupertinoPopupSurface] is composed of proxy boxes, which mimic
```

Comment:
nit: I think this could just be regular code comments, since this is justification for making this field non-nullable?

Reply:
Fixed!